assemblies/assembly-install-and-configuring-a-floating-action-button.adoc

```diff
@@ -0,0 +1,10 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: install-and-configuring-a-floating-action-button
+[id="{context}"]
+= Installing and configuring a floating action button in {product}
+
+As a {product-short} administrator, you can configure the global floating action button as a dynamic plugin by using the `global floating action button` plugin to any {product} page that you want. You have the option to create submenus in the global floating action button by configuring any number of action buttons in the same slot in your `root.tsx` file.
```

Comment:
when configuring the button as a dynamic plugin you wouldn't need to make changes in the root.tsx file
check this out https://github.com/redhat-developer/rhdh-plugins/tree/main/workspaces/global-floating-action-button/plugins/global-floating-action-button#installing-as-a-dynamic-plugin